go.mod

```diff
@@ -1,27 +1,82 @@
 module github.com/aws/amazon-vpc-resource-controller-k8s
 
-go 1.16
+go 1.20
 
 require (
 	github.com/allegro/bigcache/v3 v3.1.0
 	github.com/aws/amazon-vpc-cni-k8s v1.9.0
```

Comment:
Can you also update this to v1.12.2? I can approve after that

Reply:
@jdn5126 vpc cni version was updated. thanks